generic/tests/cfg/iometer_windows.cfg

```diff
@@ -73,3 +73,37 @@
                     bg_mode = yes
                 - after_test:
                     bg_mode = no
+        - with_multi_scsi_disks:
+            only virtio_scsi
+            run_timeout = 3600
+            icf_name = "iometer_multi_disks.icf"
+            format_multi_disks = yes
+            disk_indexes = "1 2 3 4 5 6"
```

Comment:
hi yhong, why here test 6 disks? any other requirement exist? I see the polarion case test 4 disks~

Reply:
ok, will update it, if you think 4 disks are ok for yours testing.